### ISSUE TITLE
Deprecation warning due to invalid escape sequences.

### ISSUE THREAD

**Issue by tirkarthi** [IBM/ibm-cos-sdk-python-s3transfer#5]
Deprecation warning due to invalid escape sequences. Using raw strings or escaping them again helps in resolving this. Check https://github.com/asottile/pyupgrade/ for automatic fix of this.

```
find . -iname '*.py' | grep -Ev 'rdf4|tool' | xargs -P4 -I{} python3.8 -Wall -m py_compile {}
./tests/unit/test_utils.py:303: DeprecationWarning: invalid escape sequence \.
  '%s\.[0-9A-Fa-f]{8}$' % filename,
```

**Comment by rtveitch** [comment]
Thanks for the notice, we'll take a look and get back to you.

**Comment by rtveitch** [comment]
This is fixed in the latest patch release, `2.6.3`.
